b3lb/loadbalancer/urls.py

```diff
@@ -28,5 +28,9 @@
     path('b3lb/ping', ping),
     url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/bbb/api/(?P<endpoint>[0-9.a-zA-Z]*)$', api_pass_through),
     url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/stats', stats),
-    url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/metrics', metrics)
+    url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/metrics', metrics),
+    url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/logo', logo),
+    url(r'^b3lb/t/(?P<slug>[a-z]{2,10}(-(?P<sub_id>\d{3}))?)/slide', slide),
+    # necessary for db-file-storage extension!
+    url(r'^files/', include('db_file_storage.urls'))
```

Comment:
For which use-case? Django admin? Maybe we should only pick parts of `db_file_storage.urls`? **TBD**